docs/self-managed/setup/deploy/local/manual.md

```diff
@@ -92,23 +92,37 @@ You’ll know Zeebe has started successfully when you see a message similar to t
 [exporter] [0.0.0.0:26501-zb-actors-1] INFO  io.camunda.zeebe.broker.exporter.elasticsearch - Exporter opened
 ```
 
-You can test the Zeebe Gateway by asking for the cluster topology with [zbtcl](/apis-tools/community-clients/cli-client/index.md#usage):
-
-```shell
-./bin/zbctl --insecure status
-```
-
-`zbctl status` should produce an output like this:
-
-```
-Cluster size: 1
-Partitions count: 1
-Replication factor: 1
-Gateway version: 8.1.6
-Brokers:
-  Broker 0 - 0.0.0.0:26501
-    Version: 8.1.6
-    Partition 1 : Leader, Healthy
+You can test the Zeebe Gateway by asking for the cluster topology with the [Camunda 8 REST API](/apis-tools/camunda-api-rest/camunda-api-rest-overview.md):
+
+```
+curl -L 'http://localhost:8080/v2/topology' \
+-H 'Accept: application/json'
+```
+
+Which should produce an output like this:
+
+```json
+{
+  "brokers": [
+    {
+      "nodeId": 0,
+      "host": "string",
+      "port": 0,
+      "partitions": [
+        {
+          "partitionId": 0,
+          "role": "leader",
+          "health": "healthy"
+        }
+      ],
+      "version": "8.8.0"
+    }
+  ],
+  "clusterSize": 1,
+  "partitionsCount": 1,
+  "replicationFactor": 1,
+  "gatewayVersion": "8.8.0"
```

Comment:
non-blocking: one down-side of including the full output of this specific endpoint is that the version number may look very out of date when we're on 8.10.0 and haven't updated this output. It's technically not a big deal because it _should_ output the version of your actual cluster, but it also might be a bad look for our docs to show anything multiple versions behind. 

What do you think about removing those specific details by either not emitting all fields and including `...` to suggest there are more, or replacing those values with placeholders like `<cluster version>`?

Reply:
I don't feel strongly enough about it to block you on this, so leave it as-is! 

Re: variables in a code block, I haven't looked at this at all. Maybe open an issue to investigate that capability?

Reply:
https://github.com/camunda/camunda-docs/issues/5058